src/styles/variables.scss

```
@@ -0,0 +1,7 @@
+$calendar-cell-size: 100px;
+$calendar-gap: 1px;
+$calendar-padding: 10px;
+
+$week_days: (
+  1:"mon", 2:"tue", 3:"wed", 4:"thu", 5:"fri", 6:"sat", 7:"sun"
```

```suggestion
  1:"mon", 
  2:"tue", 
  3:"wed", 
  4:"thu", 
  5:"fri", 
  6:"sat", 
  7:"sun";
```

Each value in new line, and `;` at last value